fdbserver/storageserver.actor.cpp

```diff
@@ -5967,7 +5974,7 @@ ACTOR Future<GetMappedKeyValuesReply> mapKeyValues(StorageServer* data,
 		                      pOriginalReq->options.get().debugID.get().first(),
 		                      "storageserver.mapKeyValues.BeforeLoop");
 
-	for (; offset<sz&& * remainingLimitBytes> 0; offset += SERVER_KNOBS->MAX_PARALLEL_QUICK_GET_VALUE) {
+	for (; (offset < sz) && (*remainingLimitBytes > 0); offset += SERVER_KNOBS->MAX_PARALLEL_QUICK_GET_VALUE) {
```

Comment:
This may break CI's format (clang-format v15) check.

Reply:
We'd better to update clang-format to v16 in CI. Once I save storageserver.actor.cpp, this never comes back until I manually do fmt.